src/app/(private)/(routes)/(member)/forum/components/post.tsx

```diff
@@ -60,7 +60,13 @@ const Post = ({
               src={post.user.imageUrl || avatarImg.src}
             />
             <div className="flex flex-col ml-2">
-              <span className="font-bold text-base">
+              <span
+                onClick={e => {
+                  e.stopPropagation()
+                  router.push(`/user/${post.user.username}`)
```

Comment:
Same thing here. We should have a fallback for this behavior.

Reply:
if (!post) {
    notFound()
  }